### ISSUE TITLE
ArgumentError: wrong number of arguments (9 for 8)

### ISSUE THREAD

**Issue by masterkain** [logentries/le_ruby#44]
the recent change on this line: https://github.com/logentries/le_ruby/blob/master/lib/le.rb#L30
does not match the signature of this one: https://github.com/logentries/le_ruby/blob/master/lib/le/host.rb#L4

resulting in a:

```
ArgumentError: wrong number of arguments (9 for 8)
/home/ubuntu/apps/histreet/shared/bundle/ruby/2.2.0/gems/le-2.7.0/lib/le/host.rb:5:in `new'
/home/ubuntu/apps/histreet/shared/bundle/ruby/2.2.0/gems/le-2.7.0/lib/le.rb:30:in `new'
```

**Comment by goozler** [comment]
Hello! I still use an older version 2.3.9, but I have this error too in last two days

**Comment by karlatkinson** [comment]
Getting the same issue also :+1:

**Comment by forrestgrant** [comment]
+1

**Comment by StephenHynes7** [comment]
Sorry guys my fault, IDE suggested a very similar name to what I had planned to use and I completely overlooked it. Should be fixed now in 2.7.1
